src/gradualizer_lib.erl

```diff
@@ -159,14 +164,17 @@ pick_value(?type(non_neg_integer), _Env) ->
 pick_value(?type(pos_integer), _Env) ->
     {integer, erl_anno:new(0), 0};
 pick_value(?type(neg_integer), _Env) ->
-    {integer, erl_anno:new(0), -1};
+    L = erl_anno:new(0),
+    {op, L, '-', {integer, L, 1}};
```

Comment:
Just out of curiosity, why is this an improvement?

Reply:
If we're strict, and Gradualizer is, then `abstract_expr()` doesn't allow negative numbers - we always deal with non-negatives, possibly wrapped in unary op nodes. I'm just applying feedback from Gradualizer itself - the spec says we return an `abstract_expr()`, so let's do it.